public/app/features/alerting/unified/components/receivers/TemplateDataDocs.tsx

```diff
@@ -0,0 +1,204 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data/src';
+import { Stack, useStyles2 } from '@grafana/ui/src';
+
+export function TemplateDataDocs() {
+  return (
+    <Stack gap={2}>
+      <TemplateDataTable caption="Template Data" dataItems={GlobalTemplateData} />
+      <TemplateDataTable
+        caption="Alert template data - available only when in the context of an Alert"
+        dataItems={AlertTemplateData}
+      />
```

Comment:
```suggestion
        caption="Alert template data is only available when creating an alert."
        dataItems={AlertTemplateData}
      />
```

Reply:
I am not sure entirely what you mean here. Only available when creating an alert rule / just alert? i know the two are used interchangeably and it would be helpful to decide on one and be consistent. :)

Reply:
Yep, it's quite tricky to explain it. `In the context of an alert` means that the alert object is available. You can be in the context of an alert e.g. inside alerts for loop
Outside of this loop, a single alert is not available 
```
{{ range .Alerts }}
    ---ALERT---
    Alertname: {{ toUpper .Labels.alertname }}
    StartsAt: {{ .StartsAt }}
    EndsAt: {{ .EndsAt }}
{{ end }}

Reply:
So can you say that? .... is only available in the .Alerts loop (is that official terminology and understood??)

Reply:
> is only available in the .Alerts loop (is that official terminology and understood??)
It's only one of the options. You can also define the loop in a different template file. 
There are various ways in which you can structure your templates, especially, you can nest one template inside another, and depending on that you may or may not be in the context of an Alert. 
I know it's quite difficult to understand without prior Go templating knowledge

Reply:
I added .Alerts loop as an example

Reply:
ok, always better with an example :)